superset-frontend/stylesheets/less/variables.less

```diff
@@ -190,4 +195,8 @@
 /* in favor of custom/reusable CSS wherever possible                    */
 /************************************************************************/
 
+// ***************************** SIP 34 UI *******************************
+@table-hover: rgba(236, 238, 242, 0.5);
```

Comment:
Happy to help move some of these new things over to the Theme file in superset-ui to move toward Emotion theme use. I can show you the (fairly easy) way to bump superset-ui package releases for this flow.

Reply:
would these be available in LESS files? That's currently what I'm working with. I'd suggest moving this way for now and when we do theming work later we can look into consolidating these.